lib/modules/manager/nuget/readme.md

```diff
@@ -1,5 +1,21 @@
-The `nuget` configuration object is used to control settings for the NuGet package manager.
-The NuGet package manager supports a SDK-style `.csproj`/`.fsproj`/`.vbproj`/`.props`/`.targets` format, as described [here](https://natemcmaster.com/blog/2017/03/09/vs2015-to-vs2017-upgrade/).
```

Comment:
I've removed the link from the docs, but maybe we still want it? I'd prefer to link to some _official_ docs instead.

Reply:
Let's remove